inst/include/unitval.hpp

```diff
@@ -82,6 +82,7 @@ enum unit_types {
   U_TG_NMVOC,
   U_TG_H2,
   U_DU_O3,
+  U_DU_O3_TG,
```

Comment:
What's the unit here, exactly? Might be worth an explanatory comment.

Reply:
Is it O3 Dobson Units per Tg emissions (in this case it is H2 emissions)